SU-HSDO/suhumsci
# feat(STN-83): news + horizontal card work

**In progress thoughts...**
- trying to think through whether we put styles for horizontal card pattern under patterns/ in humsci_basic or the src/scss folder
- as of right now, src/scss seems easier, but we may want to reconsider if patterns start to bloat that folder and then move them

# Summary
[JIRA card](https://sparkbox.atlassian.net/secure/RapidBoard.jspa?rapidView=80&modal=detail&selectedIssue=STN-83)

Creates the styles and HTML for the horizontal card and News Item content type. We are not worrying about events in this context yet.

Additionally, I created some SCSS functions and mixins for looking up the primary, secondary, and tertiary colors for the colorful-teal theme variation. Also, added sass-true tests.

Design: https://www.figma.com/file/toNZnFLadtpNpG5dPzNGWz/%E2%9A%99%EF%B8%8F-Colorful-Theme---Working?node-id=1408%3A1313

# Steps to Test
1. Add a news test page to your local drupal:
- On an existing site (using the colorful theme), create a few pieces of dummy “News Item” content. Remember to add some without images, one without an external link, etc.
- Create an empty “basic page.”
- On the basic page, add a paragraph:
  - View → News
    - Display: → List
- Ensure that the news items are being displayed on the page and the default looks good
(Note that layout is not created yet, so everything will be 100%)
2. inside humsci_basic theme folder, run `npm run test` and make sure everything passes
# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
